src/components/charts/HorizontalText.vue

```diff
@@ -59,5 +57,9 @@ export default {
 
 </script>
 
-<style>
+<style lang="scss" scoped>
+  .grid_8 {
```

Comment:
grid_8 is not very descriptive, why not use a descriptive class name and add this class to the div?

Reply:
added a comment